components/browser/state/src/main/java/mozilla/components/browser/state/state/SessionState.kt

```diff
@@ -13,11 +13,13 @@ package mozilla.components.browser.state.state
  * @property engineState the [EngineState] of this session.
  * @property extensionState a map of extension id and web extension states
  * specific to this [SessionState].
+ * @property contextId the session context ID of the session.
```

Comment:
nit: I feel like this could need some more information about what a context id is or what it does. I think that is completely unclear if you do not know upfront. Or is there some documentation we could link to?

Reply:
Maybe https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Work_with_contextual_identities

Reply:
Fixed.